.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.md 
+!README.md
```

Comment:
As the previous reviewer requested, please add test.md and remove README.md from gitignore file.